examples/to_mdbx.rs

```diff
@@ -0,0 +1,150 @@
+//! Convert a block folder to a MDBX folder
+//! For help, run: `cargo run --example to_mdbx -- --help`
+
+use alloy_primitives::{Address, Bytes, FixedBytes, B256, B64, U64};
+use clap::Parser;
```

Comment:
We don't want these complications this early. `data` is already hard-coded for tests & benchmarks, let's just do the same here. Practically there is little demand to run any of these manually.